packages/esm-patient-medications-app/src/api/order-config.ts

```diff
@@ -34,7 +41,7 @@ export function useOrderConfig(): {
   };
 } {
   const { data, error, isLoading, isValidating } = useSWRImmutable<{ data: OrderConfig }, Error>(
-    `${restBaseUrl}/orderentryconfig`,
+    `${restBaseUrl}/orderentryconfig?v=full`,
```

Comment:
Were you not able to get it working with the custom format @piyushmishra1416 ? If not, please report the request payload size difference between the old call and the new one.